src/parser/xlsx/utilities/common.util.ts

```diff
@@ -0,0 +1,35 @@
+import * as xlsx from 'xlsx';
+
+
+export default class CommonUtil {
+
+    /**
+     * Checks all required sheets are present in excel file or not
+     * @returns Returns boolean value for whether all required sheets are present in excel file or not
+     */
+    static validateSheetNames(requiredSheetNames: string[], availableSheetNames: string[]) {
+        return Object.values(requiredSheetNames).every((knownSheetName) => availableSheetNames.find((availableSheet: string) => availableSheet?.trim() === knownSheetName?.trim()));
```

Comment:
Please check this for the logic https://stackoverflow.com/questions/53606337/check-if-array-contains-all-elements-of-another-array